packages/components/src/menu-group/README.md

```diff
@@ -1,6 +1,31 @@
 # MenuGroup
 
-## Usage
+`MenuGroup` wraps a series of related `MenuItem` components into a common section.
+
+![MenuGroup Example](https://wordpress.org/gutenberg/files/2019/03/MenuGroup.png)
+
+1. MenuGroup
+
+## Table of Contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
+
+## Design guidelines
+
+### Usage
+
+A `MenuGroup` should be used to indicate that two or more individual MenuItems are related. When other menu items exist above or below a `MenuGroup`, the group should have a divider line between it and the adjacent item. A MenuGroup can optionally include a label to describe its contents. 
+
+![MenuGroup diagram with label and dividers](https://wordpress.org/gutenberg/files/2019/03/MenuGroup-Anatomy.png)
+
+1. MenuGroup label
```

Comment:
"MenuGroup" may be redundant, but it is a bit more clear since you mention other components above. I'll leave that up to you.

Reply:
Yeah, I think it's fine to repeat that there, for clarity's sake. Those should have tick marks like the other mentions of the components though. 👍